src/tests/win-app-driver/win-app-driver.csproj

```diff
@@ -10,7 +10,7 @@
   <ItemGroup>
     <PackageReference Include="Appium.WebDriver" Version="4.2.1" />
     <PackageReference Include="DotNetSeleniumExtras.WaitHelpers" Version="3.11.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />

```

Comment:
Any reason to not replace this with `Microsoft.JSON`?

Reply:
That project is currently not being used, but we want to get back to it to get some automated UI tests eventually. That'll be something for that time.